modules/xmpp/RtxModifier.spec.js

```diff
@@ -0,0 +1,263 @@
+/*eslint-disable max-len*/
+/*jshint maxlen:false*/
+var RtxModifier = require("./RtxModifier.js");
```

Comment:
Can we stick with ES6 "import" with the new code ? (please fix all occurrences)
In general we should try to use ES6 with the new code. I'm not sure if that works with the tests, does it ?